pkg/types/imagebased/imagebased_config_types.go

```diff
@@ -108,6 +108,10 @@ type InstallationConfig struct {
 	// PullSecret is the secret to use when pulling images.
 	PullSecret string `json:"pullSecret"`
 
+	// ReleaseRegistry is the container image registry that hosts the OpenShift release-image content.
+	// +optional
+	ReleaseRegistry string `json:"releaseRegistry,omitempty"`
```

Comment:
Please mention that this registry is used during pre-caching (while performing the preparation/installation step) and can be different than the registry to be used at the edge site deployment